benchopt/plotting/html/static/result.js

```diff
@@ -167,7 +190,7 @@ const getScatterCurves = () => {
         legendgroup: solver,
         hovertemplate: '(%{x:.1e},%{y:.1e}) <extra></extra>',
         visible: isVisible(solver) ? true : 'legendonly',
-        x: data(solver).scatter.q9,
+        x: data(solver).scatter[scatterXaxisProperties[1]],
         y: useTransformer(data(solver).scatter.y, 'y', data().transformers),
       });
```

Comment:
Maybe skip this all together if `xaxisType` is not `time`?

Reply:
Thanks for pointing this out, quantiles lose sense as ``stop_val`` is deterministic hence won't change throughout the reps.